release/Magarena/scripts/Lightning_Coils.groovy

```diff
@@ -0,0 +1,49 @@
+def trigger = new AtEndOfTurnTrigger() {
+    @Override
+    public MagicEvent executeTrigger(final MagicGame game,final MagicPermanent permanent,final MagicPlayer eotPlayer) {
+        return new MagicEvent(
+            permanent,
+            this,
+            "Exile SN."
+        );
+    }
+    @Override
+    public void executeEvent(final MagicGame game, final MagicEvent event) {
+            game.doAction(new RemoveFromPlayAction(event.getPermanent(), MagicLocationType.Exile));
+    }
+}
+
+def playMod = {
```

Comment:
no needed, use MagicPlayMod.EXILE_AT_END_OF_TURN

Reply:
fixed, the trigger above can be then also removed